File: Practices/old_enough.py
#CC 6th, Old Enough

age = int(input("How old are you? "))

if age >= 18:
    print("You are old enough to vote!")
elif age >= 16:
    print("You are old enough to drive!")
elif age >= 15:
    print("You are old enough to get a learners permit!")
elif age >= 5:
    print("You are old enough to go to school!")
else:
    print("You are not old enough to go to school.")

File: c_programming/c_programming/practices/old_enough.c
//CC 6th old enough practice
#include <stdio.h>
#include <string.h>
 int main(void){

    int age;
    char name[10000];
     printf("What is ur age??: ");
    scanf("%d", &age);

    if (age < 0) {
        printf("Please enter a real age.\n");
    } else if (age >= 18) {
        printf("You can vote!\n");
    } else if (age >= 16) {
        printf("You can drive!\n");
    } else if (age >= 15) {
        printf("You can get ur permit!\n");
    } else if (age >= 6) {
        printf("You can go to school!\n");
    } else {
        printf("You cant do any of these.\n");
    }

    return 0; 
}